### ISSUE TITLE
Consider providing a "template" jupyter notebook containing only headings/markdown

### ISSUE THREAD

**Issue by raar1** [carpentries-lab/deep-learning-intro#186]
The following high level workflow is taught:

1. Formulate/ Outline the problem
2. Identify inputs and outputs
3. Prepare data
4. Choose a pre-trained model or build a new architecture from scratch
5. Choose a loss function and optimizer
6. Train the model
7. Perform a Prediction/Classification
8. Measure Performance
9. Tune Hyperparameters
10. Share Model

Each stage is, of course, detailed and needs a lot of lesson time to explore. The workflow is followed again for different example problems. While teaching this course, we have had feedback that students sometimes lose track of the stage in the overall workflow we are at, and where they are in the grand scheme (i.e. what is coming next).

Providing a notebook template with e.g. just these headings (and maybe a little explanation under each) could help here. There would be no code in the template, only markdown, and some placeholder empty code cells in each section.

**Comment by dafnevk** [comment]
This seems like a good format to try out when teaching this lesson. If you teach it this way and it works well, feel free to share the template so that others can use it too.
